app/experimenter/targeting/constants.py

```diff
@@ -462,11 +463,30 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
-NEW_USERS_WITH_INFREQUENT_USE = NimbusTargetingConfig(
-    name="New users with infrequent use",
-    slug="new_users_with_infrequent_use",
-    description="0 - 6 days activity in past 28 days and profile age < 28 days",
-    targeting=(f"{PROFILELESSTHAN28DAYS} " "&& userMonthlyActivity|length <= 6"),
```

Comment:
I hope the above targeting is not referenced in any experiment??

Reply:
oh, I just checked that there are 4 experiments in prod 3 are in preview and 1 in draft using that targeting, I would suggest sending all back to draft first and then merging this PR, and then selecting this new targeting.
Just for future reference, we avoid removing targeting as it can be referenced in old experiments.